File: S/OOP/C++/00_access_modifiers.cpp
class Student 
{
    // 存取修飾字
    private:
        // 只有 class 內部能存取
        string name;
        int age;
        double gpa;

    public:
        // 任何人都能存取
        void setName(string n) { name = n; }
        void setAge(int a) { age = a; }
        string getName() { return name; }

    protected:
        // 只有自己和子類別能存取（繼承時用到）
        int studentId;
};

File: S/OOP/Java/00_access_modifiers.java
class Student {
    // 存取修飾字
    private String name;
    private int age;
    private double gpa;

    public void setName(String n) { name = n; }
    public void setAge(int a) { age = a; }
    public String getName() { return name; }

    protected int studentId; // 只有自己和子類別能存取（繼承時用到）
}